packages/fast-components-react-msft/src/action-toggle/action-toggle.props.ts

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import { Omit } from "utility-types";
+import {
+    ActionToggleClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-msft";
+import {
+    ButtonAppearance,
+    ButtonHandledProps,
+    ButtonManagedClasses,
+    ButtonUnhandledProps,
+} from "../button/button.props";
+
+export enum ActionToggleAppearance {
+    lightweight = ButtonAppearance.lightweight,
+    primary = ButtonAppearance.primary,
+}
+
+export interface ActionToggleManagedClasses
+    extends ManagedClasses<ActionToggleClassNameContract> {}
+export interface ActionToggleHandledProps
+    extends Omit<ButtonHandledProps, "appearance" | keyof ButtonManagedClasses>,
+        ActionToggleManagedClasses {
+    /**
+     * The action toggle selected glyph render prop
+     */
+    selectedGlyph?: (className: string) => React.ReactNode;
+
+    /**
+     * The action toggle usselected glyph render prop
+     */
+    unselectedGlyph?: (className: string) => React.ReactNode;
+
+    /**
+     * The action toggle selected content
+     */
+    selectedContent?: React.ReactNode;
+
+    /**
+     * The action toggle unselected content
+     */
+    unselectedContent?: React.ReactNode;
+
+    /**
+     * The action toggle unselected ARIA text
+     */
+    unselectedLabel: string;
```

Comment:
In the README, you call this `selectedARIALabel` but here it is `unselectedLabel`. These should be consistent